instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld

```diff
@@ -254,18 +254,18 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Papp et al. 2015.",
   "isAlternativeVersionOf": null,
   "isNewVersionOf": null,
   "keyword": null,
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"

```

Comment:
was the previous one registered incorrect or did you change it?

Reply:
We changed it on June 28, 2023.